not-supported/backup-viewer.sh

```diff
@@ -150,6 +150,14 @@ then
     fi
 fi
 
+# Check if pending snapshot is existing a second time and cancel the viewing in this case.
+if does_snapshot_exist "NcVM-snapshot-pending"
+then
+    msg_box "The snapshot pending does exist. Can currently not show the backup.
```

Comment:
Does it exist or not? :)

Reply:
You mean the snapshot?

The "snapshot-pending" does exist, hence you cannot show the backup. This is correct.
I guess the wording isn't that clear...